apiserver/facades/controller/crossmodelrelations/crossmodelrelations.go

```diff
@@ -307,8 +322,10 @@ func (api *CrossModelRelationsAPI) registerRemoteRelation(relation params.Regist
 
 // WatchRelationUnits starts a RelationUnitsWatcher for watching the
 // relation units involved in each specified relation, and returns the
-// watcher IDs and initial values, or an error if the relation units could not be watched.
-func (api *CrossModelRelationsAPI) WatchRelationUnits(remoteRelationArgs params.RemoteEntityArgs) (params.RelationUnitsWatchResults, error) {
+// watcher IDs and initial values, or an error if the relation units
+// could not be watched.  WatchRelationUnits is only supported on the
+// v1 API - later versions provide WatchRelationChanges instead.
+func (api *CrossModelRelationsAPIv1) WatchRelationUnits(remoteRelationArgs params.RemoteEntityArgs) (params.RelationUnitsWatchResults, error) {
```

Comment:
We need to mask this ob V2 right?

Reply:
No, V1 embeds V2, so methods added on V1 aren't visible to V2 - it's only new methods on V2 that need to be masked from older versions.